firmware.conf

```diff
@@ -1,8 +1,3 @@
-[ilo]
-version = 1.96
-url = https://downloads.hpe.com/pub/softlib2/software1/sc-linux-fw-ilo/p1980791503/v97349/CP023365.scexe
-file = ilo196.bin
-
```

Comment:
This is not a duplicate section, but a definition of the 'default' firmware for the ilo1. ilo2/3/4 have a similar section